scripts/mysql_system_tables.sql

```diff
@@ -205,12 +205,18 @@ CREATE TABLE IF NOT EXISTS time_zone_leap_second (   Transition_time bigint sign
 
 CREATE TABLE IF NOT EXISTS procs_priv ( Host char(60) binary DEFAULT '' NOT NULL, Db char(64) binary DEFAULT '' NOT NULL, User char(32) binary DEFAULT '' NOT NULL, Routine_name char(64) COLLATE utf8_general_ci DEFAULT '' NOT NULL, Routine_type enum('FUNCTION','PROCEDURE') NOT NULL, Grantor char(93) DEFAULT '' NOT NULL, Proc_priv set('Execute','Alter Routine','Grant') COLLATE utf8_general_ci DEFAULT '' NOT NULL, Timestamp timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP, PRIMARY KEY (Host,Db,User,Routine_name,Routine_type), KEY Grantor (Grantor) ) engine=InnoDB STATS_PERSISTENT=0 CHARACTER SET utf8 COLLATE utf8_bin   comment='Procedure privileges' TABLESPACE=mysql;
 
+-- bug#92988: Temporarily turn off sql_require_primary_key for the
+-- next 2 tables as this is not necessary as they do not replicate.
```

Comment:
PK also improve performance for non-replicated InnoDB, but this isn't InnoDB.

Reply:
Indeed.